src/components/CodeInput/PCodeInput.vue

```diff
@@ -111,6 +112,7 @@
     return {
       textarea: {
         height: `${lineHeight.value * lines.value}px`,
+        width: `${internalValue.value.length * 8 + 10}px`,
```

Comment:
This calculation is a bit ad hoc.  I'd hoped to use 'ch' but that caused my UI to crash so px was the best alternative.